packages/account/src/providers/transaction-summary/operations.ts

```diff
@@ -239,9 +239,10 @@ export function getContractCallOperations({
   abiMap,
   rawPayload,
   maxInputs,
+  baseAssetId,
```

Comment:
Would this inclusion make this a breaking change?

Reply:
@petertonysmith94 No, this helper is intended for internal use only.